### PR TITLE
version bump: 0.70.0

### PR DESCRIPTION
**Pull request by cognifloyd**
Prepare next release v0.70.0

### CODE REVIEW

**Comment by cognifloyd** [comment]
@armab what else do you do when cutting a release?

File: Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 # StackStorm version which refers to Docker images tag
 appVersion: "3.5dev"
```
Comment (cognifloyd) [inline Chart.yaml:3]: For the release should this be:
```suggestion
appVersion: "3.5"
```

Comment (armab) [reply Chart.yaml:3]: Sounds good to me!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## In Development
+
+## v0.61.0
```
[the remark this answers is not in the document]
Comment (cognifloyd) [reply CHANGELOG.md]: done

**Review by armab — APPROVED** [review]
Thanks!

The next step is to actually make the Github release, building a pushing an artifact to https://github.com/StackStorm/stackstorm-ha/tree/gh-pages
That'll make it available from https://helm.stackstorm.com/

**Pull request merged**